app/models/user.rb

```diff
@@ -20,7 +20,7 @@ class User < Common::RedisStore
   redis_store REDIS_CONFIG['user_b_store']['namespace']
   redis_ttl REDIS_CONFIG['user_b_store']['each_ttl']
   redis_key :uuid
-
+  attr_writer :identity
```

Comment:
I'm not sure we want to be able to update UserIdentity from user object, this opens up the possibility for certain things I'm weary of introducing, like modifying UserIdentity attributes after the response is received through the user object.